config/rbac/servicebinding_controller_rolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: controller-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: service-binding-operator-controller-role
```

Comment:
Where is this ClusterRole defined?

Reply:
we are adding this ClusterRole as an aggregator ClusterRole, to make it pluggable for other controllers to add additional rules. We are adding this to manifests here: https://github.com/redhat-developer/service-binding-operator/pull/979/files#diff-3fd064f2f1027a954b035a6d5885b96cc5b483f04889d14fa840ace5e9c29c55R8-R9

Reply:
but should not be the task of `kustomize` to prepend `service-binding-operator` prefix to this name?

Reply:
kustomization only changes `.metadata.name` not in the roleRef